tests/core/test_scraper.py

```diff
@@ -166,6 +166,8 @@ def test_get_ethpm_birth_block(w3, interval):
     time_travel(w3, interval)
     actual = get_ethpm_birth_block(w3, 0, w3.eth.blockNumber, latest_block.timestamp)
     assert actual == latest_block.number - 1
+    # Not found
+    assert get_ethpm_birth_block(w3, 0, 0, latest_block.timestamp) == -1
```

Comment:
Hope it's okay to add this to the existing test instead of a separate function even though it's no longer looking at only valid intervals

Reply:
It'd be better if it went in its own test - testing that it raises an exception (a la piper's comment). In most cases, it's better to separate testing concerns into as many functions as necessary rather than squeezing them into a single test (though I'm definitely guilty of this also)